src/nyc_trees/apps/survey/fixtures/species.json

```diff
@@ -1278,12 +1265,12 @@
         "model": "survey.Species",
         "fields": {
             "species_code": "PR",
-            "scientific_name": "Prunus serrulata",
+            "scientific_name": "Prunus cultivar",
```

Comment:
The ``Treecorder List`` column of ``Species_List_rev3.xlsh`` for the species with forms_id 92410 is ``Cherry - PRUNUS``. If I understand the spreadsheet correctly, the ``scientific_name`` value should be change to ``"Prunus"`` to get the desired appearance in the treecorder.